setup.cfg

```diff
@@ -32,11 +32,12 @@ keywords =
 python_requires = >=3.7
 packages = ctdcal
 install_requires =
+    cmocean
     click
     gsw >= 3.2
     matplotlib
     numpy >= 1.19.0
-    pandas >= 1.0.5
+    pandas==1.1.2  # >1.2.1 breaks GSW's ufuncs when given multiple Series inputs, see release notes
```

Comment:
Is this some bug in gsw now re pandas compatibility? Or an incompatibility with in how ctdcal is using pandas+gsw?

Reply:
My brief testing seemed to suggest it is a gsw/pandas issue. I was going to submit an issue on the GSW-Python page but wasn't quite sure if I should? Mainly because I don't have a good handle on whose bug it is, gsw or pandas.

Here's the testing I tried:
```
import gsw
import numpy as np
import pandas as pd
print(gsw.SA_from_SP([30,30,30], [1000,1000,1000], 0, 0))  # test 1
print(gsw.SA_from_SP(pd.Series([30,30,30]), [1000,1000,1000], 0, 0))  # test 2
print(gsw.SA_from_SP(pd.Series([30,30,30]), pd.Series([1000,1000,1000]), 0, 0))  # test 3
```

The first two test work with every version combination I tried, the third fails for newer pandas versions. I think [this change](https://pandas.pydata.org/docs/whatsnew/v1.2.1.html#calling-numpy-ufuncs-on-non-aligned-dataframes) is the issue, which is odd though since even doing pd.align() doesn't fix issue.

Reply:
Resolved the 3 places this was causing issue